backend/src/folder/dto/update-folder.input.ts

```diff
@@ -0,0 +1,8 @@
+import { CreateFolderInput } from './create-folder.input';
+import { InputType, Field, PartialType } from '@nestjs/graphql';
+
+@InputType()
+export class UpdateFolderInput extends PartialType(CreateFolderInput) {
+  @Field()
+  title: string;
```

Comment:
Nitpick:

Since your `UpdateFolderInput` already extends `PartialType(CreateFolderInput)` you actually don't need to to add title in here

What it does, is that` UpdateFolderInput` will already have `title` property but it will be optional (nullable)

Reply:
Done!